thorlcr/graph/thgraph.hpp

```diff
@@ -1030,7 +1030,7 @@ class graph_decl CActivityBase : public CInterface, implements IExceptionHandler
 
 // IExceptionHandler
     bool fireException(IException *e);
-    void processAndThrowOwnedException(IException * e) __attribute__((noreturn));
+    __declspec(noreturn) void processAndThrowOwnedException(IException * e) __attribute__((noreturn));
```

Comment:
Most definitions have both the windows and linux annotations - this only had linux.